Ullaakut/cameradar
# Add scan interval option

## Goal of this PR

Fixes #243 

This PR introduces a new option, `--attack-interval`, shortened as `-I`, which allows users to perform attacks on remote cameras with an interval between each attempt, in order to keep the attack hidden from network monitoring tools that are setup to detect bruteforce attacks.

It also fixes some unclear documentation about the use of duration type options, such as the `--timeout` option and the new `--attack-interval` option. It now properly describes that they are not taking an amount of milliseconds, but a duration (such as `2ms`).

## How to test it

* Ensure that increasing the interval value increases the time it takes to attack a camera.
* Ensure that by default, no interval is added.
* Ensure that negative values can not be given.

This screenshot demonstrates the feature (the time the operation took can be seen in the prompt below, 8 seconds with a 2000ms interval and 40 seconds with a 10000ms interval).

<img width="955" alt="Screenshot 2019-11-11 at 9 38 58 PM" src="https://user-images.githubusercontent.com/6976628/68619390-b16ffc80-04cb-11ea-9c3a-e59a2d1ad3b7.png">
